test/org/opensolaris/opengrok/configuration/messages/MessagesTest.java

```diff
@@ -133,15 +133,15 @@ public void testMessageSendSuccess() {
     @Test
     public void testMessageSendWrongHost() {
         RuntimeEnvironment.getInstance().setMessageLimit(100);
-        Assert.assertEquals(5, invokeMain("localhost", portNum + 2));
+        Assert.assertEquals(1, invokeMain("localhost", portNum + 2));
```

Comment:
it's the changed exit code. `Messages` produce error messages to `stderr` so I saw no extra value in the distinct error codes.